src/main/resources/messages.properties

```
@@ -259,8 +259,8 @@ onboarding-provider-info-review.info-header-2=Provider contact info
 onboarding-provider-info-review.link=Edit their info
 
 #onboarding-provider-info-confirm
-onboarding-provider-info-confirm.title=Confirm provider info
-onboarding-provider-info-confirm.header=Thanks! Now we can help you contact {0} when you''re done.
+onboarding-provider-info-confirm.title=Message to provider needed
+onboarding-provider-info-confirm.header=You will need to message your child care provider at the end
 onboarding-provider-info-confirm.notice=<div class='notice--success spacing-below-60'><p>At the end of this application, you'''ll have the option to <strong>text or email a secure link</strong> to '{0}'.</p><p>Using the link, your provider can quickly complete their part of the application.</p></div>
```

You did not need to add any more text string. All you need here is, changing the copy in there and then adding an orange div 

```suggestion
onboarding-provider-info-confirm.notice=<p>You will need to <strong>email or text a secure link</strong> to your provider so they can complete their part.</p><div class="notice--warning spacing-below-60"><p>You can't get help with child care costs until both you and your provider finish your parts of the application.</p></div>
```